examples/laser_STC/analysis_laser_STC.py

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env python3
+
+# Copyright 2025
+#
+# This file is part of HiPACE++.
+#
+# Authors: Xingjian Hui
+# License: BSD-3-Clause-LBNL
+
+import argparse
+import numpy as np
+import scipy.constants as scc
+from lasy.utils.laser_utils import get_dispersion, get_zeta, get_beta
+from lasy.laser import Laser
+from lasy.profiles import FromOpenPMDProfile
+
+lambda0 = .6e-6          # Laser wavelength
+k0 = 2 * scc.pi / lambda0
+
+parser = argparse.ArgumentParser(description='Compare laser propagation in vacuum with theory')
+parser.add_argument('--output-dir',
+                    dest='output_dir',
+                    default='diags/hdf5',
+                    help='Path to the directory containing output files')
+args = parser.parse_args()
+
+profile = FromOpenPMDProfile(
+    file_name=args.output_dir+ '/openpmd_000000.h5',
+    envelope_name = 'laserEnvelope',
+)
+
+laser = Laser(
+        dim="xyt",
+        lo=(-10e-6, -10e-6, -0e-15),
+        hi=(10e-6, 10e-6, +200e-15),
+        npoints=(255, 255, 730),
+        profile=profile,
+     )
+Phi2, phi2 = get_dispersion(laser.grid,laser.dim, omega0 = 2 * scc.pi * scc.c / lambda0, order = 2)
```

Comment:
If you're not using it
```suggestion
_, phi2 = get_dispersion(laser.grid,laser.dim, omega0=2 * scc.pi * scc.c / lambda0, order=2)
```